src/librustc_middle/mir/interpret/error.rs

```diff
@@ -452,6 +452,8 @@ pub enum UnsupportedOpInfo {
     NoMirFor(DefId),
     /// Encountered a pointer where we needed raw bytes.
     ReadPointerAsBytes,
+
+    // The variants below are only reachable from CTFE/const prop, miri will never emit them.
```

Comment:
Please leave an empty line above and below, otherwise this "melds" into the following doc-comment.

Also while you are at it, could you fix the doc-comment for the `enum` itself?

Reply:
Rustfmt eats that line. I did have that line ;)

Reply:
Seriously, rustfmt is getting annoying. :/

Maybe then format it like that:
```
//
// text
//
```
or does rustfmt also remove empty comments?

Reply:
yea that worked